ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java

```diff
@@ -31,25 +31,30 @@ public void setUp() throws Exception {
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore //ignore due to moving forms functionality
     @Test
     public void addFormTest() throws Exception {
         homePage.goToManageForm();
-        if (!manageForm.addPresent()) {
-            manageForm.delete();
-        }
+        String Add = "containing add button";
+        if(!manageForm.containsText(Add)) {
```

Comment:
any difference between this check and what manageForm.addPresent() does?

Reply:
manageForm.addPresent() wont work since addPresent method only returns when add button is present, what i did is create a string which is not available in manageFormPage .

Reply:
can't you use the same method to pick the non existing event?

Reply:
Let me try that and see thanks

Reply:
so continues to fail if using same method something like 
  _if(manage.addPresent()){
   
}_else{
}
This continues to fail ,unless we create a dummy data and check its availability, in that we get rid of using addPresent  method, will be glad for your help thanks cc @k-joseph

Reply:
I think you are interested in something like this!
```java
if(manage.addPresent()){
  // put your add form testing code in here
} else{
  // resolve/pass and proceed
}
```